src/state/publicFunding/saga.js

```diff
@@ -34,7 +35,7 @@ function getReqPublicFunding({
     end,
     limit: LIMIT,
     filter,
-    symbol: formatRawSymbols(mapRequestSymbols(targetSymbol, true)),
+    symbol: formatRawSymbols(mapRequestSymbols(targetSymbol, true), isFunding),
```

Comment:
since `isFunding` is always true, value of `symbol:` will always have`f` prefix. is this expected?

Reply:
Yes, it's expected behavior here.